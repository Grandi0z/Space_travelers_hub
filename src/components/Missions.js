import React from 'react'
import Table from 'react-bootstrap/Table';

const Missions = () => {
  return (
    <Table striped bordered hover>
      <thead>
        <tr>
          <th>Mission</th>
          <th>Description</th>
          <th>Status</th>
          <th>Unname</th>
        </tr>
      </thead>
      <tbody>
        
      </tbody>
    </Table>
  );
}

    

export default Missions