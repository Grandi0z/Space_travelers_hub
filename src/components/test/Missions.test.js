import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { configureStore } from '@reduxjs/toolkit';
import Missions from '../Missions';
import missionsReducer from '../../redux/features/mission/missionsSlice';

const store = configureStore({
  reducer: {
    missions: missionsReducer,
  },
});

describe('Missions', () => {
  it('Should match the snapshot', () => {
    const tree = renderer.create(
      <Provider store={store}><Missions /></Provider>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
