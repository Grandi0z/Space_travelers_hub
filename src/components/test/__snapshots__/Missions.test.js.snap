// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Missions Should match the snapshot 1`] = `
<div>
  No mission yet
</div>
`;
